app/controllers/util/SupervisingCustomsOfficeHelper.scala

```diff
@@ -28,21 +28,21 @@ import uk.gov.hmrc.http.HeaderCarrier
 
 object SupervisingCustomsOfficeHelper {
```

Comment:
It might be better to have this object changed to class to be able to inject it and mock in tests